docs/undergrad.yaml

```diff
@@ -506,13 +506,11 @@ Measures and integral Calculus:
     integrable functions: 'measure_theory.integrable'
     dominated convergence theorem: 'measure_theory.tendsto_integral_of_dominated_convergence'
     finite dimensional vector-valued integrable functions: 'measure_theory.integrable'
-    continuity of integrals with respect to parameters: 'interval_integral.continuous_on_integral_of_continuous'
-    differentiability of integrals with respect to parameters: 'interval_integral.differentiable_on_integral_of_continuous'
-    fundamental theorem of calculus, part 1: 'interval_integral.integral_has_fderiv_at_of_tendsto_ae'
-    fundamental theorem of calculus, part 2: 'interval_integral.integral_eq_sub_of_has_deriv_right_of_le'
+    continuity of integrals with respect to parameters:
```

Comment:
I've reverted this to its original (blank) state.  But perhaps we could count `measure_theory.continuous_integral` as giving this statement?  To quibble, I suppose that to get the classic undergrad result one also needs a continuous coercion map from `bounded_continuous_function` (on a compact space) to `measure_theory.l1`.

Reply:
No, this whole section is all about https://github.com/leanprover-community/sphere-eversion/blob/master/src/parametric_integral.lean#L88 I simply need to find time to update mathlib in this project and quickly PR all that file before Yury or Sébastien refactor integration once more.